Wiley Online Library.js

```diff
@@ -255,6 +255,11 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				item.title = ZU.capitalizeTitle(item.title, true);
 			}
 			
+			//date in the cochraine library RIS is wrong
+			if (ZU.xpathText(doc, '//meta[@name="citation_book_title"]/@content') == "The Cochrane Library") {
```

Comment:
Though even if the date is not present, I think it's probably better that the user enters it manually, since we know that it's almost certainly wrong from the RIS file.